data_structures/src/data_request.rs

```diff
@@ -384,134 +316,85 @@ pub fn calculate_dr_vt_reward(dr_output: &DataRequestOutput) -> u64 {
 pub fn calculate_tally_change(dr_output: &DataRequestOutput, n_reveals: u64) -> u64 {
     calculate_reveal_reward(dr_output) * (u64::from(dr_output.witnesses) - n_reveals)
 }
-
+/////////// TODO: Posible Remove //////////////
```

Comment:
We will probably remove it later during the VRF implementation, for now we can keep it.